src/app/wallet/v2/views/asset_list.nim

```diff
@@ -1,5 +1,6 @@
-import NimQml, Tables
+import NimQml, Tables, trait_list
 from status/wallet2 import OpenseaAsset
+import NimQml, Tables, trait_list
```

Comment:
we usualy seperate our import from std import. So maybe add a new import line with trait_list

Reply:
Done!